<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerDialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_receivedMessages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>202</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>451</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sendMessage">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>115</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Send Message</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_receiver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>0</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Broadcast</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
